src/widgets/mfa/trustedDevicesWidget.tsx

```diff
@@ -0,0 +1,141 @@
+                    <button className="p-[calc(var(--padding-y)*1px)]"><Delete className="fill-dangerColor w-[2vw] h-[2vh]"/></button>
+                </AlertDialog.Trigger>
+                <AlertDialog.Portal>
+                    <AlertDialog.Overlay/>
+                    <AlertDialog.Content
+                        className="fixed left-1/2 top-1/2 max-h-[85vh] w-[90vw] max-w-[500px] -translate-x-1/2 -translate-y-1/2 rounded-md bg-gray1 p-[25px] shadow-[var(--shadow-6)] focus:outline-none">
+                        <AlertDialog.Title className="m-0 text-[17px] font-medium">
+                            {i18n('trustDevice.delete.confirmation')}
+                        </AlertDialog.Title>
+                        <div className="flex justify-end gap-[3vw] pt-[1vw]">
+                            <AlertDialog.Cancel asChild>
+                                <button className="transition ease-in-out hover:scale-110 fill-white px-[1vh] shadow-lg hover:bg-gray-200 rounded-sm">{i18n('confirmation.cancel')}</button>
+                            </AlertDialog.Cancel>
+                            <AlertDialog.Action asChild>
+                                <button onClick={_ => onDelete(device)} className="transition ease-in-out hover:scale-110 hover:bg-red-500 px-[1vh] bg-dangerColor shadow-lg text-white rounded-sm">{i18n('confirmation.yes')}</button>
+                            </AlertDialog.Action>
+                        </div>
+                    </AlertDialog.Content>
+                </AlertDialog.Portal>
+            </AlertDialog.Root>
+        )
+    }
+
+
+    return (
+        <div>
+            {(trustedDevices.length === 0) && (
+                <Info>{i18n('trustedDevices.empty')}</Info>
+            )}
+            {trustedDevices.map((trustedDevice, index) => (
+                <div id={`trusted-device-${index}`} key={`trusted-device-${index}`} className={`flex items-center ${isOpen ? 'opacity-15' : ''}`}>
+                    <div className="flex flex-col items-center basis-full line-height-1 align-middle whitespace-nowrap box-border p-[calc(var(--padding-y)*1px)] border-solid border-[calc(var(--border-width)*1px)] rounded-[calc(var(--border-radius)*1px)]" >
+                        <div className="font-bold ">{trustedDevice.metadata.deviceName}</div>
+                        <div className="font-bold">{trustedDevice.metadata.ip}</div>
+                        <div className="font-bold">{trustedDevice.metadata.deviceClass}</div>
+                        <div className="font-bold">{trustedDevice.metadata.operatingSystem}</div>
+                        <div className="font-bold">{trustedDevice.metadata.userAgent}</div>
+                        <div>
+                            <span>{i18n('mfaList.createdAt')}&nbsp;: </span>
+                            <time
+                                dateTime={trustedDevice.createdAt}>{dateFormat(trustedDevice.createdAt, config.language)}</time>
```

Comment:
```suggestion
                                dateTime={trustedDevice.createdAt}>{intlFormat(new Date(trustedDevice.createdAt), { locale: config.language })}</time>
```
(sans oublier `import { intlFormat } from 'date-fns';`